### PR TITLE
Update the accessibility guidelines page

### PR DESCRIPTION
**Pull request by lyubomir-popov**
## Done

Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/3046

- Remove some of the aspirational stuff, as it sounds somewhat patronising (others may disagree)
- Focus on what we have actually done in recent memory, rather than preaching
- Update the layout a bit

## QA

- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/accessibility or [demo]
- Read the text, see if you agree with the changes
- Provide feedback :)

### CODE REVIEW

**Comment by bartaz** [comment]
The external link icon drops to new line in last section:

Screenshot 2020-10-09 at 09 19 46

We should probably look into fixing this in the link, but maybe we can reword the sentence here to make sure it doesn't happen by default?

File: templates/accessibility.html [hunk 1]
```diff
@@ -5,71 +5,82 @@
 {% block copydoc %}https://docs.google.com/document/d/1_cCvuHSwS9i0pzD_4WHDFoTenVAfZVr9qGxG-rSHHGY/edit{% endblock %}
```
Comment (bartaz) [inline templates/accessibility.html:5]: This page has a copydoc: https://docs.google.com/document/d/1_cCvuHSwS9i0pzD_4WHDFoTenVAfZVr9qGxG-rSHHGY/edit#heading=h.k45g1u10i0ni

I think we should update that as well to the new content.

File: templates/accessibility.html [hunk 2]
```diff
@@ -5,71 +5,82 @@
...
+    </div>
+    <div class="col-6">
+      <p class="p-heading--5 u-no-margin--bottom"><a href="https://www.w3.org/WAI/eval/report-tool/#!/evaluation/audit#%2F" class="p-link--external">Accessibility report tool</a></p>
+      <p>A checklist that can be filtered by A / AA / AAA level, with a short description and links to the related "Understanding" and "How to Meet" articles that accompany each criterion.</p>
+      <p class="p-heading--5 u-no-margin--bottom"><a href="https://www.w3.org/TR/wai-aria-practices" class="p-link--external">WAI-ARIA Authoring Practices 1.1</a></p>
+      <p>
+        <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr>A guide for understanding how to use <cite><a href="https://www.w3.org/TR/wai-aria-1.1/"><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> 1.1</a></cite> to create an accessible Rich Internet Application. It provides guidance on the appropriate application of <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr>, describes recommended <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> usage patterns, and explains concepts behind them.</p>
```
Comment (bartaz) [inline templates/accessibility.html:25]: Isn't there one A too much in first acronym? 

Screenshot 2020-10-09 at 09 26 23

**Comment by lyubomir-popov** [comment]
Thakns @bartaz fixed the typo and dropping external link icon. 
@anthonydillon can you please take a look at these changes and let me know if you agree?

**Review by bartaz — APPROVED** [review]
LGTM

**Review by anthonydillon — APPROVED** [review]
This is great content. Thanks @lyubomir-popov

**Comment by anthonydillon** [comment]
This section looks a little disconnected by I see its in flow with the rest of the page. Maybe this section could just be text below title? 
image

**Comment by lyubomir-popov** [comment]
Thanks @anthonydillon, I've updated the text to sit under the heading.

image

**Pull request merged**